linkcheckerrc

```diff
@@ -40,6 +40,7 @@ ignore=
   file:///*
   http://che.aws.my-ide.cloud
   https://che.openshift.io/*
```

Comment:
Should it be removed?

Reply:
yup, but there are still some links that point to openshift.io - https://github.com/eclipse/che-docs/pull/1866/files